aerospike-mixin/README.md

```diff
@@ -57,6 +57,13 @@ scrape_configs:
           instance: "<your-instance-name>"
           aerospike_cluster: "<your-cluster-name>"
           __path__: /var/log/aerospike/aerospike.log
+    pipeline_stages:
+        - multiline:
+            firstline: '\w{3} \d{2} \d{4}'
+        - regex:
+            expression: '\w{3} \d{2} \d{4} \d{2}:\d{2}:\d{2} \w{3}: (?P<level>\w+) (?P<context>\(\w+\)): (?P<trace>\(\S+\))\s+(?P<message>.*)'
+        - labels:
+            level:
 ```
 
 ## Aerospike Namespace Overview
```

Comment:
## Aerospike namespace overview?

Reply:
Sorry, could you clarify the meaning of this comment? I've moved the log config to be under the logs dashboard and updated the regex, let me know if there's anything else.

Reply:
Sentence case should be used everywhere in Grafana where applicable. 
https://grafana.com/docs/writers-toolkit/write/style-guide/capitalization-punctuation/#sentence-case-capitalization